cwf/cloud/go/services/cwf/obsidian/models/validate.go

```diff
@@ -45,7 +44,7 @@ func (m *GatewayCwfConfigs) ValidateModel(context.Context) error {
 	for _, peer := range m.AllowedGrePeers {
 		for _, key := range set[peer.IP] {
 			if swag.Uint32Value(peer.Key) == key {
-				return errors.New(fmt.Sprintf("Found duplicate peer %s with key %d", peer.IP, key))
+				return fmt.Errorf("Found duplicate peer %s with key %d", peer.IP, key)
```

Comment:
Noticed that the PR description says this would be substituted with errors.New from std lib. I don't think this way might be nicer, since you don't need the Sprintf. But may be adapt the PR summary?

Reply:
I agree. Also, I stumbled across a few cases like this in #12724, where there was a `Sprintf` inside an `errors.New`. When we just change to `errors.New` from the std lib, the linter will complain and suggest using `fmt.Errorf` instead, as it is done here.
But yes, we should add this case to the summary.

Reply:
I updated the summary.